packages/protocol/contracts/common/MetaTransactionWallet.sol

```diff
@@ -201,22 +209,36 @@ contract MetaTransactionWallet is
    * @param values The CELO value to be sent with each transaction.
    * @param data The concatenated data to be sent in each transaction.
    * @param dataLengths The length of each transaction's data.
+   * @return The return values of all transactions appended as bytes and an array of the length
+   *         of each transaction output which will be 0 if a transaction had no output 
    */
   function executeTransactions(
```

Comment:
just had a random thought
we could have 2 versions of this function or a wrapper function which saves on gas by not building the return values array
lmk if you think that would actually be useful or if the gas discrepancy is insignificant

Reply:
with today's ethereum prices, it would add 10 cents onto the transaction in our test (which is fairly big). So obv .....half the gas price 5 cents.  Lol where can I find the celo gas prices at the moment?? I think it's fiine,  but maybe a feature we can add if it's needed?

Reply:
sgtm